part1-장현태-sprint4/src/repositories/articleRepository.js

```diff
@@ -0,0 +1,105 @@
+import prisma from "../config/prisma.js";
+
+async function getAll(query) {
```

Comment:
getAll이라는 함수명은 데이터를 전부 가져올 것 같은 의미가 느껴집니다. 그래서 아래와 같이 수정하면 어떨까 싶습니다.
```suggestion
async function getMany(query) {
```
or
```suggestion
async function getFiltered(query) {
```